odyssey.conf

```diff
@@ -356,6 +356,7 @@ database default {
 #		"clear_text" - PostgreSQL clear text authentication
 #		"md5"        - PostgreSQL MD5 authentication
 #		"cert"       - Compare client certificate Common Name against auth_common_name's
+#		"pam"        - Use PAM service
```

Comment:
I do not think this is correct to introduce authentication method here. It should be done the same way as `auth_query` handled and should work both for `clear_text` and `md5` methods, otherwise it gets a bit confusing